content/docs/start/data-management/data-pipelines.md

```diff
@@ -47,6 +47,10 @@ $ tree
     ├── prepare.py
     ├── requirements.txt
     └── train.py
+
+$ git init  # if needed
+$ dvc init
+$ git commit .dvc params.yaml src
```

Comment:
Ah wait I forgot about fetch.
```suggestion
$ git commit -a -m "Create evaluation stage"
$ dvc fetch data/data.xml
```
Or should it be pull?
```suggestion
$ git commit -a -m "Create evaluation stage"
$ dvc pull data/data.xml
```
Please commit the right one.